src/scss/components/_donors.scss

```diff
@@ -0,0 +1,61 @@
+.donors_info {
+  @include grid;
+}
+
+.donors {
+  position: relative;
+  overflow-x: hidden;
```

Comment:
I would suggest to use:
```
overflow: hidden;
```
to remove double vertical scrollbar

Reply:
thanks, i corrected it!!